oak_runtime/src/introspect.rs

```diff
@@ -95,6 +95,42 @@ fn find_ids(path: &str, kind: &str) -> Option<(u64, u64)> {
     Some((id, subid))
 }
 
+// Looks for a matching file generated by webpack and returns it
+fn find_client_file(path: &str) -> Option<(File, String)> {
+    let filepath = Regex::new(r"^/dynamic/(?P<filepath>[^\s]+)$")
+        .unwrap()
+        .captures(path)?
+        .name("filepath")
+        .unwrap()
+        .as_str();
+
+    // Do not serve paths that go up the tree
```

Comment:
Just to check: will all possible encodings of "../" be encompassed by this?  (I *think* all URL encodings will be gone by now, but I'm not 100%)

Reply:
It seems hyper already takes care of decoding, and infact will remove any instances `../` by capping it to the root path.

Eg requesting `/dynamic/../../graph.rs` will yield "/graph.rs" as the path. Requests with encoded URLs are refused by the server.